wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
 
-A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned."
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since most skins do away with showing that a player has hit a note perfectly (to only show the minimum amount of information to a player in-game). 
+- Another gameplay-improving feautre that skins provide (for osu! standrad) is chagning the cursor style and trail. This can be especially helpful if a player finds the default skin cursor intruding into the action.
+
+## How to Use Custom Skins
+
+Skins can either be created from scratch (see [Skinning](/wiki/Skinning) for creating your own skin), or downloaded as a ".osk" file and imported into the game. To download and use an osu! skin you have to:
+
+1. Pick one of the [Popular Skin Websites](#popular-skin-websites) and open the link.
+2. Choose a skin that suits your tastes by clicking on it's name or picture.
```

Comment:
This depends on the site a user is visiting and should not be something that needs to be explained here

Reply:
> This depends on the site a user is visiting and should not be something that needs to be explained here

I'll just add a small description before the numbered lists containing a link to the (Skinning 
Guides and Import Threads article)[https://github.com/ppy/osu-wiki/blob/master/wiki/Skinning/Guides_and_important_threads/en.md] for the reader to acquire their own skin with guides methods and collections.

> The completed skins compendium, as well as the skinning forum in general should be added here.

The article also houses links to the important forums and the completed skins compendium.